scripts/package

```diff
@@ -9,13 +9,29 @@ SUFFIX=""
 
 cd $(dirname $0)/..
 
-mkdir -p dist/artifacts rootfs
-cd rootfs
-
-mkdir -p usr/bin
-
-cp ../bin/per-host-subnet usr/bin/per-host-subnet
-
-tar cvzf ../dist/artifacts/rancher-per-host-subnet-${ARCH}-${VERSION}.tar.gz .
-
-echo "Built rancher-per-host-subnet-${ARCH}-${VERSION}.tar.gz"
+package (){
+    mkdir -p dist/artifacts rootfs
+    pushd rootfs
+
+    if [ "$1" == "linux" ]; then
+        mkdir -p usr/bin
+        cp ../bin/$1/$2/per-host-subnet usr/bin/per-host-subnet
+        tar cvzf ../dist/artifacts/rancher-per-host-subnet-${ARCH}-${VERSION}.tar.gz .
+        echo "Built rancher-per-host-subnet-${ARCH}-${VERSION}.tar.gz"
+    else
+        mkdir -p rancher
+        cp ../bin/$1/$2/per-host-subnet.exe rancher/per-host-subnet.exe
+        cp ../startup_per-host-subnet.ps1 rancher/startup_per-host-subnet.ps1
+        zip -r ../dist/artifacts/rancher-per-host-subnet.zip rancher/
+        echo "Built ranchet-per-host-subnet.zip"
+    fi
+    popd
+
+}
+
+if [ "$CROSS" == "true" ]; then
```

Comment:
@orangedeng can you update this to `-n "$CROSS" ` like it was updated in rancher/agent so it will produce the correct `.zip` files?